src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -227,6 +229,16 @@ def setUp(self):
         if self.nvme_parameter == "nvme":
             server_utils.storage_prepare(self.hostlist_servers)
 
+            # Clean daos mnt
+            self.assertTrue(dmg_utils.clean_daos_mnt(self.hostlist_servers))
+
+            # Format hostlist for dmg tool
+            port = self.params.get("port", "/run/server_config/*")
```

Comment:
How this is going to work? We need to start the server as root to initiate the daos_shell (storage_format) command. Is this test code working for you in local environment?